modules/data.land/R/soilgrids_soc_extraction.R

```diff
@@ -0,0 +1,209 @@
+##' soilgrids_soilC_extract function
+##' A function to extract total soil organic carbon for a single or group of 
+##' lat/long locationsbased on user-defined site location from SoilGrids250m 
+##' version 2.0 : https://soilgrids.org
+##' @title soilgrids_soilC_extract
+##' @name soilgrids_soilC_extract
+##' 
+##' @param site_info A dataframe of site info containing the BETYdb site ID, 
+##' site name, latitude, and longitude, e.g. 
+##' (site_id, site_name, lat, lon)
+##' @param outdir Optional. Provide the results as a CSV file 
+##' (soilgrids_soilC_data.csv)
+##' @param verbose Provide progress feedback to the terminal? TRUE/FALSE
+##' 
+##' ##' @examples
+##' \dontrun{
+##' 
+##' # Example 1 - using the modex.bnl.gov BETYdb and site IDs to extract data
+##' db <- 'betydb'
+##' host_db <- 'modex.bnl.gov'
+##' db_port <- '5432'
+##' db_user <- 'bety'
+##' db_password <- 'bety'
+##' 
+##' bety <- list(user='bety', password='bety', host='modex.bnl.gov',
+##' dbname='betydb', driver=RPostgres::Postgres(),write=FALSE)
+##' 
+##' con <- DBI::dbConnect(drv=bety$driver, dbname=bety$dbname, host=bety$host, 
+##' password=bety$password, user=bety$user)
+##' 
+##' suppressWarnings(site_qry <- glue::glue_sql("SELECT *, ST_X(ST_CENTROID(geometry)) AS lon,
+##' ST_Y(ST_CENTROID(geometry)) AS lat FROM sites WHERE id IN ({ids*})",
+##' ids = c("676","622","678","766","764"), .con = con))
+##' 
+##' suppressWarnings(qry_results.1 <- DBI::dbSendQuery(con,site_qry))
+##' suppressWarnings(qry_results.2 <- DBI::dbFetch(qry_results.1))
+##' DBI::dbClearResult(qry_results.1)
+##' dbDisconnect(con)
+##' 
+##' site_info <- qry_results.2
+##' verbose <- TRUE
+##' system.time(result_soc <- soilgrids_soilC_extract(site_info=site_info, verbose=verbose))
+##' result_soc
+##' 
+##' }
+##' @importFrom reshape2 melt
+##' @importFrom terra vect project rast extract
+##' @importFrom stats qgamma optim
+##' @importFrom utils flush.console setTxtProgressBar txtProgressBar
```

Comment:
This is OK to do, but in general the PEcAn norm is to use the namespaces in the code itself rather than using `@importFrom`, which we generally restrict for instances where namespaces don't make sense (e.g. extended tidyverse pipes)

Reply:
Okay! They occurred after the "make document" command. How can I get rid if them?

Reply:
These lines we're commenting on (in the Roxygen comment block) are the source of the ones that appeared after running `make`. If you delete these and then rerun `make document`, the corresponding `importFrom` lines in NAMESPACE should disappear.